### PR TITLE
Search results v2

### PR DESCRIPTION
**Pull request by ricardoom**
Not sure if this is ready for prime time... but the branch is getting a bit crazy..

### CODE REVIEW

File: docs/css/example-styles.css [hunk 1]
```diff
@@ -3390,7 +3414,7 @@ button:focus {
   font-style: italic; }
 
 .nypl-item-info {
-  border-left: 0.5rem solid #776e64;
+  border-left: 0.5 solid #776e64;
```
Comment (ricardoom) [inline docs/css/example-styles.css]: the `rem` value needs putting back...

File: docs/css/example-search-styles.css [hunk 2]
```diff
@@ -3390,7 +3414,7 @@ button:focus {
   font-style: italic; }
 
 .nypl-item-info {
-  border-left: 0.5rem solid #776e64;
+  border-left: 0.5 solid #776e64;
```
Comment (ricardoom) [inline docs/css/example-search-styles.css]: the `rem`value needs putting back here too...

Comment (EdwinGuzman) [inline docs/css/example-search-styles.css]: This is the css file. So did it get overwritten or something when compiling?

File: docs/discovery-item.v2.html [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+          <input type="submit" id="nypl-omni-button" value="Search">
+            <!-- <svg xmlns="http://www.w3.org/2000/svg" width="15" height="19" viewBox="0 0 15.73646 19.35436" preserveAspectRatio="xMidYMid meet" aria-labelledby="search-icon" role="img" aria-hidden="true">
+              <title id="search-icon">search</title>
+              <path d="M.35944,19.1961a.85555.85555,0,0,0,1.19271-.2011l3.81409-5.36107A7.21,7.21,0,1,0,3.93352,12.697L.15834,18.00339A.85553.85553,0,0,0,.35944,19.1961ZM8.5473,12.668a5.47881,5.47881,0,1,1,5.47881-5.47881A5.485,5.485,0,0,1,8.5473,12.668Z"/>
+            </svg> -->
+        </fieldset>
+      </form>
+
+      <div class="nypl-row search-control">
+          <svg xmlns="http://www.w3.org/2000/svg" width="25" height="42" viewBox="0 0 25 42" preserveAspectRatio="xMidYMid meet" aria-labelledby="left-arrow" aria-hidden="true">
```
Comment (EdwinGuzman) [inline docs/discovery-item.v2.html]: Maybe the SVG should be part of the `<a>`?

File: docs/discovery-item.v3.html [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+
+          <dt>Alternative titles</dt>
+          <dd>
+            <ul>
+              <li>Chinese Muslims and the global Ummah : Islamic revival and ethnic identity among the Hui of Qinghai Province</li>
+              <li>Chinese Muslims and the global Ummah :</li>
+            </ul>
+          </dd>
+        </dl>
+        <aside class="place-hold-control inline">
```
Comment (EdwinGuzman) [inline docs/discovery-item.v3.html]: Is this the right element to use in this case? I thought it was just for sidebar usages?

**Comment by EdwinGuzman** [comment]
Looks good to me. Just minor comments that doesn't stop the PR from being merged.

File: docs/discovery-item.v3.html (hunk 4 above)
Comment (ricardoom) [inline docs/discovery-item.v3.html]: Indeed. the documentation can be interpreted in a way that supports my pattern: 
http://html5doctor.com/understanding-aside/
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside
and cue the actual spec:
https://www.w3.org/TR/html5/sections.html#the-aside-element

at any rate, @seanredmond asked that I approach it differntly, see that in the v.4 version, embedding the table in the `<dl>`

File: docs/discovery-item.v2.html (hunk 3 above)
Comment (ricardoom) [inline docs/discovery-item.v2.html]: I should remove this part from the PR as its not quite resolved

**Comment by ktp242** [comment]
in `example-search-styles.v2.scss`, there are some comments with more than two `//`. Is it done intentionally? Also a block of codes is commented off, maybe should just remove it?

File: sass/_forms.scss [hunk 5]
```diff
@@ -337,7 +337,7 @@ $form-padding: 0.5rem;
       padding: 0;
     }
   }
-}
+} // /omnisearch
```
Comment (ktp242) [inline sass/_forms.scss]: is this the right pattern for commenting through the toolkit repo?

**Comment by ktp242** [comment]
Despite some minor comments, it looks nice generally.

File: sass/_forms.scss (hunk 5 above)
Comment (ricardoom) [inline sass/_forms.scss]: that notation is my personal style choice 
`// /closes-the-block`
similar to an HTML closing block `</tag>` 
sometimes the nested braces get very hard to see / read and know at a glance where the block ends...

**Comment by ablwr** [comment]
Omnisearch and Omnisearch 2 examples look the same to me?

**Pull request merged**